orttraining/orttraining/python/training/ortmodule/_inference_manager.py

```diff
@@ -86,7 +86,9 @@ def forward(self, *inputs, **kwargs):
             if self._skip_check.is_set(_SkipCheck.SKIP_CHECK_BUILD_GRADIENT) is False or \
                 not self._onnx_models.exported_model:
                 # Exporting module to ONNX for the first time
+                random_states = _utils.get_random_states()
```

Comment:
Since this is common code between training manager and inference manager, may we move it inside `self._export_model`?